### PR TITLE
Feat/nested layout

### PR DESCRIPTION
**Pull request by Perry26**
Implement nested lay-out

The inner lay-out is a radial lay-out, outer lay-out doesn't do much (yet). Michel suggested to try a tree layout on the outer layer

Bugs are still present when dragging nodes (not that is was bug-free anyway), But since the inner layout fixes the nodes in a circle, we might want to reconsider the expansion/contraction behaviour of the nodes.

Though there is still some work to do, it would be great if this could be merged to prevent further conflicts down the line.

(this also contains a bugfix for the collision force)

### CODE REVIEW

File: svelte/src/routes/v2/scripts/draw/index.ts [hunk 1]
```diff
@@ -316,6 +332,11 @@ export function draw(
 		//@ts-ignore
 		d3
 			.zoom()
+			.extent([
```
Comment (christojeffrey) [inline svelte/src/routes/v2/scripts/draw/index.ts]: this should no longer be here right? @Adityapnn811

**Review by christojeffrey — COMMENTED** [review]
otherwise seems fine

File: svelte/src/routes/v2/scripts/draw/index.ts (hunk 1 above)
Comment (Adityapnn811) [reply svelte/src/routes/v2/scripts/draw/index.ts]: Yes I removed it

**Review by Adityapnn811 — CHANGES_REQUESTED** [review]
Don't forget about the zoom

**Comment by Perry26** [comment]
Should be fixed

**Review by Adityapnn811 — APPROVED** [review]

**Pull request merged**